src/PSDMatrices.jl

```diff
@@ -45,9 +48,7 @@ function todense(M::PSDMatrix)
     return M.R' * M.R
 end
 
-function X_A_Xt(; A::PSDMatrix, X::AbstractMatrix)
-    return PSDMatrix(A.R * X')
-end
+X_A_Xt(; A::PSDMatrix, X::AbstractMatrix) = PSDMatrix(A.R * X')
```

Comment:
Was the rationale behind this change that one-liners are easier to read if anonymous?

Reply:
Oh that one slipped in. Personally I find this form easier to read. Do you have a preference?

Reply:
> Do you have a preference?


Nope.